packages/date-time/src/components/CalendarDayGrid/CalendarGridDayCell.tsx

```diff
@@ -201,6 +203,11 @@ export const CalendarGridDayCell: React.FunctionComponent<ICalendarGridDayCellPr
     }
   };
 
+  let ariaLabel = dateTimeFormatter.formatMonthDayYear(day.originalDate, strings);
+  if (shouldMarkDay) {
+    ariaLabel = ariaLabel + ', ' + strings.dayMarkedAriaLabel;
```

Comment:
Would comma sign make sense in all the localizations? 
Should we maybe just have a space between two labels?

Reply:
I thought about having the space but having the comma forces narrator to have a pause between the phrases. We have at least one other instance of that in this control (formatMonthDayYear), and I haven't seen any localization issues raised with it yet